modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,182 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="replacing-a-bare-metal-control-plane-node_{context}"]
+= Replacing a bare-metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} control plane node.
+
+[IMPORTANT]
+====
+If you reuse the `BareMetalHost` objects from an existing control plane host, do not leave the `externallyProvisioned` field set to `true`.
+
+Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by {product-title}.
```

Comment:
Not sure if you missed [this comment](https://github.com/openshift/openshift-docs/pull/43915#discussion_r847697613), or if I'm just wrong about what `{product-title}` expands to. If the former then this should be something like:

```suggestion
Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by the {product-title} installer.
```

Reply:
I believe you're correct. That said, I'll request another peer review, as this module has changed quite a bit since they last saw it.

Reply:
Updated to "the {product-title} installation program", per rh style guidelines.